### DISCUSSION TITLE
How can a custom UPF IP be exposed?

### DISCUSSION THREAD

**Discussion by ggomez048** [herlesupreeth/docker_open5gs#287] [Q&A] [answered]
I'm running Open5GS on Kubernetes and I want to use a gNB set up on another PC but the IP advertised on the NGAP messages to make the GTPU tunnel is the IP of the UPF pod. How can I change that IP to an IP accessible for the gNB?

Thank you.

**Comment by herlesupreeth** [comment, marked as answer]
You can set in .env file `UPF_ADVERTISE_IP` to the IP you want your gNB to see. Once you have modified .env file you need to run `source .env` and re-deploy 5GC from that terminal

**Reply by ggomez048** [reply]
Thank you for your response, but I'm using these Helm Charts:
[https://gradiant.github.io/5g-charts/open5gs-ueransim-gnb.html](url)

There is no .env file in this installation.

**Reply by herlesupreeth** [reply]
Its showing me 404 Error if I press on that link. Also, I would suggest you raise the issue in that repo since you are using helm charts rather than this repo.

Eitherway you can set whatever IP you want your gNB to see in upf.yaml at the following location in configuration file

https://github.com/herlesupreeth/docker_open5gs/blob/9c034db839a76c31821c752f93b92793d2659bce/upf/upf.yaml#L18
